File: official/guides/customs-guide/node/create-international-shipment.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const shipment = await client.Shipment.create({
    to_address: {
      name: 'Tim Canterbury',
      company: 'Wernham Hogg',
      street1: '118 Clippenham Lane',
      city: 'Slough',
      country: 'GB',
      zip: 'SL15BE',
    },
    from_address: {
      company: 'EasyPost',
      street1: '417 Montgomery Street',
      street2: '5th Floor',
      city: 'San Francisco',
      state: 'CA',
      zip: '94104',
      phone: '415-528-7555',
    },
    parcel: {
      length: 9,
      width: 6,
      height: 3,
      weight: 20,
    },
    customs_info: {
      eel_pfc: 'NOEEI 30.37(a)',
      customs_certify: true,
      customs_signer: 'Steve Brule',
      contents_type: 'merchandise',
      contents_explanation: '',
      restriction_type: 'none',
      restriction_comments: '',
      customs_items: [
        {
          description: 'T-shirts',
          quantity: 1,
          weight: 5,
          value: 10,
          hs_tariff_number: '123456',
          origin_country: 'US',
        },
      ],
    },
  });

  console.log(shipment);
})();

File: official/guides/customs-guide/csharp/create-international-shipment.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            EasyPost.Parameters.Shipment.Create parameters = new()
            {
                ToAddress = new EasyPost.Parameters.Address.Create
                {
                    Name = "Tim Canterbury",
                    Company = "Wernham Hogg",
                    Street1 = "118 Clippenham Lane",
                    City = "Slough",
                    Country = "GB",
                    Zip = "SL15BE"
                },
                FromAddress = new EasyPost.Parameters.Address.Create
                {
                    Company = "EasyPost",
                    Street1 = "417 Montgomery Street",
                    Street2 = "5th Floor",
                    City = "San Francisco",
                    State = "CA",
                    Zip = "94104",
                    Phone = "415-528-7555"
                },
                Parcel = new EasyPost.Parameters.Parcel.Create
                {
                    Length = 9,
                    Width = 6,
                    Height = 3,
                    Weight = 20
                },
                CustomsInfo = new EasyPost.Parameters.CustomsInfo.Create
                {
                    EelPfc = "NOEEI 30.37(a)",
                    CustomsCertify = true,
                    CustomsSigner = "Steve Brule",
                    ContentsType = "merchandise",
                    ContentsExplanation = "",
                    RestrictionType = "none",
                    CustomsItems =
                    [
                        new EasyPost.Parameters.CustomsItem.Create
                        {
                            Description = "T-shirt",
                            Quantity = 2,
                            Weight = 11,
                            Value = 23,
                            HsTariffNumber = "654321",
                            OriginCountry = "US"
                        }
                    ]
                }
            };

            EasyPost.Models.API.Shipment shipment = await client.Shipment.Create(parameters);

            Console.WriteLine(JsonConvert.SerializeObject(shipment, Formatting.Indented));
        }
    }
}
